htdsa/client.py

```diff
@@ -59,7 +60,15 @@ def _uri(self, args=None):
 		return uri
 	
 	def _request(self, method, args, kwargs):
-		result = self.pool.request(method, self._uri(args), data=kwargs, **self.options)
+		uri = self._uri(args)
+		data = kwargs
+		if method == 'GET' and len(kwargs) > 0:
+			data = dict()
+			qs = urllib.parse.urlencode(kwargs)
+			uri += '?' if '?' not in uri else '&'
+			uri += qs
```

Comment:
This could probably do to be a single append-assignment instead of two.  You could even save the temporary assignment of `qs`.